CDP4CrossViewEditor.Tests/Assemblers/CrossviewPMeanArrayAssemblerTestFixture.cs

```diff
@@ -0,0 +1,701 @@
+            Assert.IsTrue(double.TryParse(arrayAssembler.ContentArray[6, 14].ToString(), out _));
+
+        }
+
+        [Test]
+        public void VerifyThatAssemblerCalculatesPMeanIfRedundancyTypeIsExternal()
+        {
+            var crossviewSheetRowAssembler = new CrossviewSheetRowAssembler();
+            crossviewSheetRowAssembler.Assemble(this.elementDefinitions);
+            this.excelRows.AddRange(crossviewSheetRowAssembler.ExcelRows);
+
+            var P_redundancy = this.elementDefinitions.FirstOrDefault()?.Parameter.FirstOrDefault(p => p.ParameterType.ShortName == "P_redundancy");
+            (this.elementDefinitions.FirstOrDefault()?.Parameter)?.Remove(P_redundancy);
+            (this.elementDefinitions.FirstOrDefault()?.Parameter)?.Add(this.pRedundancyActiveExternalParameter);
+
+            var arrayAssembler = new CrossviewArrayAssembler(this.excelRows, this.parameterTypes);
+
+            // The that power related parameters are present into the header
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("redundancy"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_on"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_stby"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_duty_cyc"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_mean"));
+
+            Assert.AreEqual(@"Crossview_ED_1.P_duty_cyc\PS_1", arrayAssembler.NamesArray[6, 11]);
+            Assert.AreEqual(@"Crossview_ED_1.P_duty_cyc\PS_1", arrayAssembler.NamesArray[6, 12]);
+            Assert.AreEqual(@"Crossview_ED_1.P_mean\PS_1", arrayAssembler.NamesArray[6, 13]);
+            Assert.AreEqual(@"Crossview_ED_1.P_mean\PS_1", arrayAssembler.NamesArray[6, 14]);
+
+            Assert.IsNotNull(arrayAssembler.ContentArray[6, 13]);
+            Assert.IsNotNull(arrayAssembler.ContentArray[6, 14]);
+
+            Assert.IsTrue(double.TryParse(arrayAssembler.ContentArray[6, 13].ToString(), out _));
+            Assert.IsTrue(double.TryParse(arrayAssembler.ContentArray[6, 14].ToString(), out _));
+        }
+
+        [Test]
+        public void VerifyThatAssemblerNotCalculatesPMeanIfPDutyCycleIsNotProperlySet()
+        {
+            var crossviewSheetRowAssembler = new CrossviewSheetRowAssembler();
+            crossviewSheetRowAssembler.Assemble(this.elementDefinitions);
+            this.excelRows.AddRange(crossviewSheetRowAssembler.ExcelRows);
+
+            Assert.IsNotNull(this.elementDefinitions.FirstOrDefault());
+
+            var P_duty_cyc = this.elementDefinitions.FirstOrDefault()?.Parameter.FirstOrDefault(p => p.ParameterType.ShortName == "P_duty_cyc");
+            (this.elementDefinitions.FirstOrDefault()?.Parameter)?.Remove(P_duty_cyc);
+
+            var arrayAssembler = new CrossviewArrayAssembler(this.excelRows, this.parameterTypes);
+
+            // The that power related parameters are present into the header
```

Comment:
Typo in comment "the that power related..."?

Given that this is repeated to many times, I would just remove this comment completely.